components/TextEditorContainer.tsx

```diff
@@ -0,0 +1,36 @@
+﻿import React, {MutableRefObject, useRef, useState} from "react";
+import Editor from "@monaco-editor/react";
+
+type EditorContentGetter = (
+    () => string
+);
+
+function TextEditorContainer() {
+    const [isEditorReady, setIsEditorReady] = useState(false);
+    const getContentOfEditorIfMounted: MutableRefObject<any> = useRef(null);
+
+    function handleIsEditorMounted(_getContentOfEditorIfMounted: EditorContentGetter) {
```

Comment:
This one can just be `getEditorContent` - this function is only called when the editor is mounted.

Reply:
Changed as per suggestion

Reply:
I think that's the wrong way round...